pkgs/development/interpreters/python/regcpython/3-10.nix

```
@@ -0,0 +1,31 @@
+{ config, lib, pkgs, ... }:
```

```suggestion
{ config, lib }:
```
Also please format with the new formatter.